generic/tests/cfg/jumbo.cfg

```diff
@@ -8,16 +8,16 @@
         mtu = 1500
     e1000:
         Linux:
-            mtu = 16110
+            mtu = 9000
```

Comment:
And if you have strong reason to adopt 9000,
why not make it as a default value and declare it below "type = jumbo"?